Yatzy-Kata/Yatzy.Core/ComputerStrategies/ComputerStrategy.cs

```diff
@@ -0,0 +1,55 @@
+using Yatzy_Kata.Data;
+using Yatzy_Kata.Interfaces;
+
+namespace Yatzy_Kata.Strategies;
+
+public class ComputerStrategy : IStrategy
+{
+    private List<int> _currentDiceRoll;
+    private readonly ComputerDiceRollStrategy _diceRollStrategy;
+    private bool _abandoned;
+    private TurnResults? _turnResults;
+
+    public ComputerStrategy(IRandom randomDiceRoll)
+    {
+        _abandoned = false;
+        _currentDiceRoll = new List<int>();
+        _diceRollStrategy = new ComputerDiceRollStrategy(randomDiceRoll);
+    }
+    
+    public TurnResults? GetTurnResults()
+    {
+        return _turnResults;
+    }
+    
+    public void CalculateTurnResults(List<Category> remainingCategories)
```

Comment:
great use of command query separation! 🎉

Reply:
Thanks!